node_modules/oae-principals/tests/test-delete.js

```diff
@@ -18,14 +18,15 @@ var assert = require('assert');
 
 var AuthzAPI = require('oae-authz');
 var AuthzUtil = require('oae-authz/lib/util');
+var DisableUsersMigration = require('../../../etc/migration/disable_users_from_tenancy/lib/disable-users-by-tenancy');
 var FollowingTestUtil = require('oae-following/lib/test/util');
+var PrincipalsDelete = require('oae-principals/lib/delete');
+var PrincipalsTestUtil = require('oae-principals/lib/test/util');
+var Redis = require('oae-util/lib/redis');
 var RestAPI = require('oae-rest');
 var SearchTestUtil = require('oae-search/lib/test/util');
 var TestsUtil = require('oae-tests');
 
-var PrincipalsDelete = require('oae-principals/lib/delete');
```

Comment:
These were in a separate block as they are dependencies from the same module so I think that was correct actually?